Sprint 10. Homework

DoNatPanic opened the pull request:
Привет! 
Прошу проверить ПР. Но у меня есть 1 вопрос:

В критериях успешной сдачи есть такой пункт:

> - Inflate выполняется в конструкторе ViewHolder-класса.

Я не совсем поняла, почему он должен быть в ViewHolder, а не в Adapter. Так как в теории на курсе **inflate** вызывался в методе **onCreateViewHolder** класса **Adapter**

DoNatPanic committed:
Sprint 10. Homework

i-masloed commented on app/src/main/res/layout/activity_search.xml:
```diff
@@ -38,5 +38,10 @@
+        android:id="@+id/recyclerView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"></androidx.recyclerview.widget.RecyclerView>
```
app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"/>

DoNatPanic replied:
Мне не хватило пояснений к этой рекомендации..
Я не поняла, к чему было приведен этот кусок кода

i-masloed commented on app/src/main/res/layout/track_view.xml:
```diff
@@ -0,0 +1,66 @@
+        android:layout_marginVertical="@dimen/card_image_vertical_margin"
+        android:layout_marginStart="@dimen/card_horizontal_margin"
+        android:elevation="0dp"
+        app:cardCornerRadius="@dimen/album_image_radius">
```
тоже хороший способ сделать скругление улов, но можн также через Glide

DoNatPanic replied:
Применила скругление через Glide

i-masloed commented:
Я не совсем поняла, почему он должен быть в ViewHolder, а не в Adapter. Так как в теории на курсе inflate вызывался в методе onCreateViewHolder класса Adapter - возможны оба варианта

DoNatPanic commented:
Привет, Ирина!

Спасибо за ревью
По замечаниям: я сделала плэйсхолдеры для обложек альбомов, они выглядят как карточки серого цвета. Просто я в макете фигмы не увидела какого-то конкретного изображения для него и поэтому решила импровизировать.

Еще ты написала, что есть рекомендации, не влияющие на результат проверки работы, но я их что-то не могу найти в ПР...

DoNatPanic committed:
Update TrackViewHolder.kt

DoNatPanic commented:
Указала плэйсхолдер в классе TrackViewHolder

i-masloed commented:
<img width="334" alt="image" src="https://github.com/DoNatPanic/playlist-maker/assets/135275336/005a9225-2fb2-44a1-9928-ee2376735a42">

DoNatPanic committed:
Update album_image.xml

DoNatPanic committed:
Cornes radius of album image with Glide

- Применила свойство centerCrop и радиус скругления картинки через Glide
- Убрала элемент Card из разметки, который раньше отвечал за скругление
- Убрала стиль для элемента Card из файла styles.xml

DoNatPanic commented:
Спасибо за ревью, хоть на него ушло больше времени чем ожидалось.
С конечными правками стало лучше чем было
